configuration/concepts/lab_tests.csv

```diff
@@ -17,7 +17,7 @@ e1dc0b09-e7c4-4f0b-9463-58aed51fa06e,,CIEL:699,O+,O+,O+,O+,,,Misc,N/A,,,,,,,,,,,
 508c9da4-56f9-4129-9a0e-766fb8e74f3e,,CIEL:1643,Malaria,Malaria,Malaria,Malaria,,,LabTest,Coded,,Positif;Negatif;Indetermine;Zero;Un plus;Deux plus;Trois plus;Quatre plus,,,,,,,,,
 7036964e-e177-465e-8af8-b1b761f290c5,,CIEL:855,Erythrocyte Sedimentation Rate,Vitesse de sedimentation,Erythrocyte Sedimentation Rate,Vitesse de sedimentation,,,LabTest,Numeric,,,,TRUE,mm/h,0,20,,,,
 a60b7de5-d126-4b28-8efc-2a81de5165f0,,CIEL:306,Widal,Widal,Widal,Widal,,,LabTest,Text,,,,,,,,,,,
-92f1d76a-4091-4b99-83d4-633b5fbd096e,,,H. Pylori,H. Pylori,H. Pylori,H. Pylori,,,LabTest,Coded,,Positif;Negatif;Indetermine,,,,,,,,,
+92f1d76a-4091-4b99-83d4-633b5fbd096e,,,H. Pylori (Sang),H. Pylori (Sang),H. Pylori (Sang),H. Pylori (Sang),,,LabTest,Coded,,Positif;Negatif;Indetermine,,,,,,,,,

```

Comment:
You've changed the product to be "H. Pylori (Sang)" but there doesn't seem to be the corresponding change in the Odoo config Haiti. Is that intended?

Reply:
It's all good now
![hps](https://github.com/mekomsolutions/openmrs-config-haiti/assets/67123286/8adc4eeb-785b-41a7-900a-86cdcaf187d0)